src/Month/index.js

```diff
@@ -50,8 +50,8 @@ export default class Month extends PureComponent {
         day = row[k];
 
         const date = getDateString(year, month, day);
-
-        if (day < 0 || day > getDaysInMonth(date)) {
+        const daysInMonth = getDaysInMonth(new Date(year, month, 1));
```

Comment:
That seems like a non obvious API. I would expect a call to getDaysInMonth to return the number of days in the supplied date's month and not force the caller to synthesize a fake date at the first of the month. Can getDaysInMonth change to do the right thing?

Reply:
`getDaysInMonth` is part of a 3rd party lib. I will look to see what other options are available.

Reply:
@drewherbst I am synthesizing a date because I need to have something to compare against to ensure that the day provided is actually a valid day in the month and not padding.